docs/CreateEdit.md

```diff
@@ -2330,15 +2330,16 @@ const PostEdit = () => (
 
 ### Grouping Inputs
 
-Sometimes, you may want to group inputs in order to make a form more approachable. You may use a [`<TabbedForm>`](#the-tabbedform-component), an [`<AccordionForm>`](#the-accordionform-component) or you may want to roll your own layout. In this case, you might need to know the state of a group of inputs: whether it's valid or if the user has changed them (dirty/pristine state).
+Sometimes, you may want to group inputs in order to make a form more approachable. You may use a [`<TabbedForm>`](#the-tabbedform-component), an [`<AccordionForm>`](#the-accordionform-component) or you may want to roll your own layout. In this case, you might need to know the state of a group of inputs: whether it's valid or if the user has changed them (dirty/touched state).
```

Comment:
I wanted to remove the pristine reference, but I don't know if the change is misleading

Reply:
No, `pristine` equivalent is `!isDirty`

Reply:
Where final-form was returning multiple boolean for the same thing, hook-form only return one